CHANGELOG/CHANGELOG-0.3.md

```diff
@@ -4,12 +4,55 @@ Changes since `v0.2.1`:
 
 ### Features
 
-- Upgrade the `config.kueue.x-k8s.io` API version from `v1alpha1` to `v1alpha2`. `v1alpha1` is no longer supported.
-  `v1alpha2` includes the following changes:
-  - Add Namespace to propagate the namespace where kueue is deployed to the webhook certificate.
-  - Add InternalCertManagement with fields Enable, WebhookServiceName and WebhookSecretName.
-  - Remove EnableInternalCertManagement. Use InternalCertManagement.Enable instead.
+- Support for kubeflow's MPIJob (v2beta1)
+- Upgrade the `config.kueue.x-k8s.io` API version from `v1alpha1` to `v1beta1`. `v1alpha1` is no longer supported.
+  `v1beta1` includes the following changes:
+  - Add `namespace` to propagate the namespace where kueue is deployed to the webhook certificate.
+  - Add `internalCertManagement` with fields `enable`, `webhookServiceName` and `webhookSecretName`.
+  - Remove `enableInternalCertManagement`. Use `internalCertManagement.enable` instead.
+- Upgrade the `kueue.x-k8s.io` API version from `v1alpha2` to `v1beta1`.
+  `v1alpha2` is no longer supported.
+  `v1beta1` includes the following changes:
+  - `ClusterQueue`:
+    - Immutability of `spec.queueingStrategy`.
+    - Refactor `quota.min` and `quota.max` into `nominalQuota and `borrowingLimit`.
+    - Swap hieararchy between `resources` and `flavors`.
+    - Group flavors and resources into `spec.resourceGroups` to make
+      co-dependent resources explicit.
+    - Move `admission` from `spec` to `status`.
+    - Add `conditions` field to `status`.
+  - `LocalQueue`:
+    - Add `admitted` field in `status`.
+    - Add `conditions` field to `status`.
+  - `Workload`:
+    - Add `metadata` to `podSet` templates.
+  - `ResourceFlavor`:
+    - Introduce `spec` to hold all fields.
+    - Rename `labels` to `nodeLabels`.
+    - Rename `taints` to `nodeTaints`.
+- Obtain queue names from label `kueue.x-k8s.io/queue-name`. The annotation with
+  the same name is still supported, but it's now deprecated.
+- Multiplatform support for `linux/amd64` and `linux/arm64`.
+- Validating webhook for `batch/v1.Job` validates kueue-specific labels and
+  annotations.
+- Use of Server-Side-Apply for updating conditions.
+- Sequential admission of jobs https://kueue.sigs.k8s.io/docs/tasks/setup_sequential_admission/
+- Preemption within ClusterQueue and cohort https://kueue.sigs.k8s.io/docs/concepts/cluster_queue/#preemption
+- Library for integrating job-like CRDs (controller and webhooks) https://sigs.k8s.io/kueue/pkg/controller/jobframework
```

Comment:
limitRange https://github.com/kubernetes-sigs/kueue/issues/541 should be included.

Reply:
Also support scheduler leader election for HA.

Reply:
Added LimitRange